### PR TITLE
metadata/install-qa-check.d: add 60libtool-la (check for unnecessary …

### PR DESCRIPTION
**Pull request by thesamesam**
….la files)

Signed-off-by: Sam James <sam@gentoo.org>

### CODE REVIEW

**Comment by mgorny** [comment]
Well, to be honest I think it'd be better if you check for files with matching names, i.e. complain for every `.la` that doesn't come with matching `.a`.

File: metadata/install-qa-check.d/60libtool-la [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	# Iterate over all the .la files we are installing to verify there's
+	# a corresponding .a file - they're pointless without a corresponding
+	# static library.
+	local file
+	local dir
+	local base
+	local bad_files=()
+	for file in "${files[@]}" ; do
+		dir=$(dirname ${file})
+		base=$(basename ${file})
```
Comment (mgorny) [inline metadata/install-qa-check.d/60libtool-la:18]: Don't call external tools when you can do it via variable substitution.

File: metadata/install-qa-check.d/60libtool-la [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		base=${base/.la/}
```
Comment (mgorny) [inline metadata/install-qa-check.d/60libtool-la:19]:
```suggestion
		base=${base%.la}
```

You don't expect `.la` in middle of the name, do you?

**Review by mgorny — CHANGES_REQUESTED** [review]

**Comment by thesamesam** [comment]
Merged a little while ago.

**Pull request closed without merging**